### PR TITLE
Get stories

### PR DESCRIPTION
**Pull request by Gimenz**
a simple feature to get contact stories

Tanpa Judul

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -1178,6 +1178,19 @@ class Client extends EventEmitter {
...
         return blockedContacts.map(contact => ContactFactory.create(this.client, contact));
     }
+
+    /**
+     * Get stories
+     * @returns {Promise<Message[]>}
+     */
+    async getStories() {
+        const messages = await this.pupPage.evaluate(async () => {
+            const models = window.Store.Msg._models.filter(x => x.__x_id.remote.server == 'broadcast');
```
Comment (tuyuribr) [inline src/Client.js:1188]: Can we filter per contact?

File: example.js [hunk 2]
```diff
@@ -40,6 +40,18 @@ client.on('message', async msg => {
...
+    } else if (msg.body.startsWith('!getstory ')) {
+        // let number = await msg.getMentions()[0]
+        const stories = await client.getStories();
+        let number = msg.body.split(' ')[1];
+        number = number.includes('@c.us') ? number : `${number}@c.us`;
+        let filtered = stories.filter(v => v.author == number);
+        msg.reply(`found ${filtered.length} stories`);
+        
+        for (let i of filtered) {
+            await sleep(2000);
```
Comment (tuyuribr) [inline example.js:52]: Change this

**Review by tuyuribr — CHANGES_REQUESTED** [review]
index.d.ts is missing

File: example.js (hunk 2 above)
Comment (Gimenz) [reply example.js:52]: omgg, i forgot to delete sleep func

did you mean filter per contact by adding this example into getStories function?

Comment (tuyuribr) [reply example.js]: Can we use something like getStories('551199991234@c.us') ?

**Review by purpshell — CHANGES_REQUESTED** [review]
use the StatusV3 module to do all this, you aren't forced to use Msg

**Comment by Koceng404** [comment]
bro kalo biar chat Groupnya Ke filter Gimana ya? jadi semua Fitur hanya bisa dipake kalo ga di chat di group?

**Review by purpshell — CHANGES_REQUESTED** [review]
You can still just get all models without specifying an id though right? just `.map()` on `_models` and you should be good

File: example.js (hunk 2 above)
Comment (PurpShell) [reply example.js]: yes, and no number returns everything.

**Pull request closed without merging**